feature/memo/src/main/java/com/memozi/memo/MemoSearchScreen.kt

```diff
@@ -0,0 +1,184 @@
+package com.memozi.memo
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.WindowInsets
+import androidx.compose.foundation.layout.asPaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.navigationBars
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.memozi.component.textfield.MemoziSearchTextField
+import com.memozi.component.top.MemoziBackground
+import com.memozi.designsystem.MemoziTheme
+import com.memozi.designsystem.R
+import com.memozi.memo.model.MemoItem
+import com.memozi.memo.model.dummyMemoItems
+
+@Composable
+fun MemoItemCard(
+    memoTitle: String,
+    memoContent: String,
+    memoDate: String,
+) {
+    Card(
+        modifier =
+            Modifier
+                .fillMaxWidth(),
+        shape = RoundedCornerShape(8.dp),
+        elevation = CardDefaults.cardElevation(4.dp),
+    ) {
+        Column(
+            modifier =
+                Modifier
+                    .fillMaxWidth()
+                    .background(Color.White)
+                    .padding(8.dp),
+        ) {
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                verticalAlignment = Alignment.CenterVertically,
+                horizontalArrangement = Arrangement.SpaceBetween,
+            ) {
+                Row(verticalAlignment = Alignment.CenterVertically) {
+                    Image(
+                        painter = painterResource(id = R.drawable.ic_diary_feed_vertical_line_black),
+                        contentDescription = null,
+                    )
+                    Spacer(modifier = Modifier.width(8.dp))
```

Comment:
```suggestion
               Text(
                text = "|",
                style = MemoziTheme.typography.ngReg15,
                color = MemoziTheme.colors.gray07,
            )
            Spacer(modifier = Modifier.width(4.dp))
```
이렇게 바꾸면 피그마 디자인에 좀 더 맞는 디자인이 될거같아요